src/components/UserManagement/UserTableData.jsx

```diff
@@ -190,9 +204,9 @@ const UserTableData = React.memo(props => {
               {DELETE}
             </button>
           </span>
-          <span className="usermanagement-actions-cell">
-            <ResetPasswordButton authEmail={props.authEmail} user={props.user} isSmallButton />
-          </span>
+          {isOwner? <span className="usermanagement-actions-cell">
+             <ResetPasswordButton authEmail={props.authEmail} user={props.user} isSmallButton    />
+           </span>:null}
```

Comment:
```suggestion
          <span className="usermanagement-actions-cell">
             <ResetPasswordButton authEmail={props.authEmail} user={props.user} isSmallButton    />
           </span>
```
Also not about creating users